stores/identity.ts

```diff
@@ -78,8 +125,13 @@ export const useIdentityStore = defineStore('identity', {
         balance: DEFAULT_BALANCE_STATE,
         tokens: emptyObject<BalanceMap>(),
       }
+      this.resetMultipleBalances()
       localStorage.removeItem('kodaauth')
     },
+    resetMultipleBalances() {
+      Vue.set(this.multiBalances, 'address', '')
+      Vue.set(this.multiBalances, 'chains', {})
```

Comment:
Have you tried `Object.assign` instead of `Vue.set`?

Reply:
let me try with `Object.assign`

Reply:
seems like didn't work, maybe related to the tip section in this docs? https://pinia.vuejs.org/core-concepts/state.html#state

Reply:
updated, yes changed to something like that works